### PR TITLE
hot-fix: fix-message-image-is-blink-in-chat

### PR DESCRIPTION
**Pull request by sherlockvn**
### Blocker:
https://github.com/linagora/matrix-dart-sdk/pull/9

### Before:
https://github.com/linagora/twake-on-matrix/assets/43041967/e3ccd2d8-e989-4f38-b4ed-064a69960d61

### After:

https://github.com/linagora/twake-on-matrix/assets/43041967/e24ebf51-10bb-4096-ad6b-20dd625bcd30

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: lib/pages/chat/events/message_content.dart [hunk 1]
```diff
@@ -420,3 +396,51 @@ class _ButtonContent extends StatelessWidget {
     );
   }
 }
+
+class MessageImageBuilder extends StatelessWidget {
```
Comment (drminh2807) [inline lib/pages/chat/events/message_content.dart:400]:
```suggestion
class _MessageImageBuilder extends StatelessWidget {
```

Comment (sherlockvn) [reply lib/pages/chat/events/message_content.dart]: done

File: lib/pages/chat/events/send_image_info_widget.dart [hunk 2]
```diff
@@ -5,8 +5,8 @@ import 'package:flutter/material.dart';
 import 'package:linagora_design_flutter/colors/linagora_ref_colors.dart';
 import 'package:matrix/matrix.dart' hide Visibility;
 
-class SendImageInfoWidget extends StatelessWidget {
-  SendImageInfoWidget({
+class SendingImageWidget extends StatelessWidget {
```
Comment (hoangdat) [inline lib/pages/chat/events/send_image_info_widget.dart:8]:
```suggestion
class SendingImageInfoWidget extends StatelessWidget {
```

File: lib/data/network/upload_file/file_info_extension.dart [hunk 3]
```diff
@@ -13,4 +13,21 @@ extension FileInfoExtension on FileInfo {
         'mimetype': mimeType,
         'size': fileSize,
       });
+
+  String get msgType {
+    return msgTypeFromMime(mimeType);
+  }
+
+  static String msgTypeFromMime(String mimeType) {
```
Comment (hoangdat) [inline lib/data/network/upload_file/file_info_extension.dart:21]:
```suggestion
  String msgTypeFromMimeType(String mimeType) {
```

**Review by hoangdat — APPROVED** [review]

**Pull request merged**